assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.js

```diff
@@ -132,15 +132,15 @@ function DashboardUniqueVisitorsWidget() {
 	}
 
 	const { totals } = visitorsData[ 0 ].data;
-	const totalUsers = totals[ 0 ].values;
-	const previousTotalUsers = totals[ 1 ].values;
-	const totalUsersChange = changeToPercent( previousTotalUsers, totalUsers );
+	const totalUsers = totals[ 0 ].values[ 0 ];
+	const previousTotalUsers = totals[ 1 ].values[ 0 ];
```

Comment:
Maybe something else changed? That _does_ look odd to me, but it appeared to work correctly in my testing. @kostyalmm can you check why you made this change and what it relates to?

Reply:
Prior to these changes, the following was happening
```
{ `${ datapoint }${ datapointUnit }` } ==> { `${ [ 10 ] }${ '%' }` } ==> 10%
``` 
and 
```
numberFormat( Math.abs( change ), {} ) ==> numberFormat( Math.abs( [ -1 ] ), {} ) ==> numberFormat( 1, {} )
```

If we pass the array after these changes, passing array will probably work fine but there will be violation of propTypes. That is why I sent the first element instead of the array.